.github/workflows/update-providers-ci-mgmt.yml

```diff
@@ -0,0 +1,38 @@
+name: Update Providers with new bridge version via ci-mgmt
```

Comment:
Perhaps this workflow can live in ci-mgmt repo?

Reply:
Ah never mind, I see this dispatches from release to update-bridge-all-providers.yml...

I think it would be good to first test out if update-bridge-all-providers.yml works, and narrow it down to ecosystem-only repos.